modules/ROOT/pages/dataweave-xml-reference.adoc

```diff
@@ -5,7 +5,7 @@ endif::[]
 :keywords: studio, anypoint, esb, transform, transformer, format, aggregate, rename, split, filter convert, xml, json, csv, pojo, java object, metadata, dataweave, data weave, datamapper, dwl, dfl, dw, output structure, input structure, map, mapping
 :page-aliases: 3.9@mule-runtime::dataweave-xml-reference.adoc
 
-Instead of using the Studio UI to add a xref:dataweave.adoc[DataWeave] script to your Mule projects, you can write a xref:6@studio::transform-message-component-concept-studio.adoc[Transform Message component] in XML, either through the XML tab in Studio or with an external editor. In fact, certain advanced features (such as the `mode` attribute  used for memory management) are only configurable through the XML.
+Instead of using the Studio UI to add a xref:index.adoc[DataWeave] script to your Mule projects, you can write a xref:6@studio::transform-message-component-concept-studio.adoc[Transform Message component] in XML, either through the XML tab in Studio or with an external editor. In fact, certain advanced features (such as the `mode` attribute  used for memory management) are only configurable through the XML.
```

Comment:
```suggestion
Instead of using the Studio UI to add a DataWeave script to your Mule projects, you can write a xref:6@studio::transform-message-component-concept-studio.adoc[Transform Message component] in XML, either through the XML tab in Studio or with an external editor. In fact, certain advanced features (such as the `mode` attribute  used for memory management) are only configurable through the XML.
```